src/app/oauth2/middleware.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
 
       req.session.shared_claims = shared_claims;
       req.session.passportSessionId = passportSessionId;
+      req.session["hmpo-wizard-cri-passport-front"] = {
```

Comment:
Is this being used anywhere if not can we follow the pattern of the 2 variables above?

Reply:
I'm not sure about this. Seamus and I had similar wonderings because it hasn't been used anywhere else.